ranger/container/fsobject.py

```diff
@@ -221,17 +222,18 @@ def set_mimetype(self):
             self._mimetype = ''
         # pylint: enable=attribute-defined-outside-init
 
-        self.video = self._mimetype.startswith('video')
-        self.image = self._mimetype.startswith('image')
-        self.audio = self._mimetype.startswith('audio')
+        self.video = self._mimetype.startswith('video/')
+        self.image = self._mimetype.startswith('image/')
+        self.audio = self._mimetype.startswith('audio/')
         self.media = self.video or self.image or self.audio
-        self.document = self._mimetype.startswith('text') \
+        self.document = self._mimetype.startswith('text/') \
```

Comment:
This seems unnecessary because the official top-level types don't have any such collisions. Have you actually encountered a problem related to this?